### ISSUE TITLE
Slack mark as read - keyboard focus

### ISSUE THREAD

**Issue by pandruszkow** [ramboxapp/community-edition#2327]
Steps to reproduce

1. Receive a message on Slack. Unread message notification appears in Rambox tab bar.
2. Switch to Slack tab by clicking on it
3. Press Esc to mark all new Slack messages as read

### Expected behavior

All unread messages are marked as read

### Actual behavior

Nothing happens.

To make this work, the following sequence of steps is required:

1. Receive a message on Slack. Unread message notification appears in Rambox tab bar.
2. Switch to Slack tab by clicking on it
3. Click anywhere *inside* the already-opened page within the Slack tab to give it keyboard focus
4. Press Esc to mark all new Slack messages as read

### ENVIRONMENT

> Rambox: v0.6.9

> OS: Windows 10

> Arch: x64

Note that a similar issue was raised in #1491 but that issue was closed due to inactivity

**Comment by Corchoneitor** [comment]
Hi @pandruszkow.

Happens as you described. There are some considerations we must attend about tab focus to fix this issue though.

Thanks!

**Comment by pandruszkow** [comment]
Hi @Corchoneitor, what kind of considerations do you mean? I'm a bit curious about this.

**Comment by elliotvhill** [comment]
@Corchoneitor I'm also having this issue with multiple apps (Gmail, Google Calendar, and Slack as well). I'm unable to use any app-specific keyboard shortcuts without following the steps @pandruszkow described in their original post.

I'm on mac os Mojave version 10.14.5 (18F132), Rambox Version: 0.6.9, Platform: darwin (x64), Electron: 4.1.4, Chromium: 69.0.3497.128, Node: 10.11.0.

Any updates on this or estimates for when a fix will be issued?
